hoodie-client/src/main/java/com/uber/hoodie/index/HBaseIndex.java

```diff
@@ -234,7 +234,8 @@ private Connection getHBaseConnection() {
 
     @Override
     public boolean rollbackCommit(String commitTime) {
-        // TODO (weiy)
+        // Can't really rollback here. HBase only can let you go from recordKey to fileID,
```

Comment:
We could potentially take snapshots and restore hbase to snapshot before commit started. But this has to be understood well before implementing